apps/src/applab/designElements/elementUtils.js

```diff
@@ -120,4 +120,13 @@ module.exports.isIdAvailable = function(newId, originalId) {
   }
 
   return true;
+};
+
+module.exports.isContentEditable = function (element) {
+  return element && element.tagName.toUpperCase() === 'DIV' && element.contentEditable;
+};
+
+module.exports.isTextInput = function (element) {
+  return element && element.tagName.toUpperCase() === 'INPUT' &&
+    (!element.type || element.type.toUpperCase() === 'TEXT');
```

Comment:
Could probably create a fairly quick and dirty mochaTest that drags each of our design element types into the visualization, the validates that isContentEditable/isTextInput does the right thing for each.